pkg/cloudprovider/providers/aws/aws_test.go

```diff
@@ -14,12 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+//go:generate mockery -dir $GOPATH/src/github.com/UKHomeOffice/keto/vendor/github.com/aws/aws-sdk-go/service/cloudformation/cloudformationiface -name=CloudFormationAPI
```

Comment:
Worth adding some comment above to highlight that those two commented lines are required and shouldn't be removed?